docs/contributing/testing.md

```diff
@@ -37,7 +37,7 @@ You should add an example to the review app if the existing examples do not refl
 
 If you've created a new component, create a new `src/govuk/<COMPONENT>/<COMPONENT>.yaml` file instead, where `<COMPONENT>` is the name of the component you've created.
 
-Use `hidden: true` in an example if you want to include the example in our [test fixtures](http://frontend.design-system.service.gov.uk/testing-your-html/) but not the review app.
+Use `hidden: true` in your example if you do not want to include the example in the review app. The example will still appear in our [test fixtures](http://frontend.design-system.service.gov.uk/testing-your-html/).
```

Comment:
Would it be helpful to say that they would probably want the example to appear in the review app if there's a visual element to it?

Reply:
Is that already covered in the first line of this section maybe? "You should add an example to the review app if the existing examples do not reflect the changes you've made."

Reply:
We should actually update that line too as not all examples appear in the review app with this new change. Maybe something along the lines of:
> You should add a new example if the existing examples do not reflect the changes you've made.

And then 

> If the example doesn't involve a visual change you can use `hidden: true` in your example to hide the example in the review app.

Reply:
Ah I see what you mean. I'm thinking this through... is it likely that an external contributor would add an example to the yaml file that would need `hidden: true`? Or is it mainly us that would be doing that? 

I'm just wondering in retrospect whether to get into `hidden: true`, because including it might mean turning this section upside-down (because it's currently primarily focused on the review app).

Reply:
I think the rest of this section is still good in terms of the references to the review app. 

If we added the above suggestions and the contributor isn't sure what we mean by 'visual change' they could ask us.

Reply:
Thanks Hanna - I've rejigged in 99267dc following your feedback and a chat with @36degrees. Hope it all makes sense now.